test/govspeak_link_extractor_with_website_root_test.rb

```diff
@@ -0,0 +1,28 @@
+require "test_helper"
+
+class GovspeakLinkExtractorWithWebsiteRootTest < Minitest::Test
```

Comment:
I would be tempted to just merge this in with the other tests for the link extractor.